dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp

```diff
@@ -642,8 +646,7 @@ void DAGStorageInterpreter::buildLocalStreams(DAGPipeline & pipeline, size_t max
         if (region_num == 0)
             continue;
         /// calculate weighted max_streams for each partition, note at least 1 stream is needed for each partition
-        size_t current_max_streams = table_query_infos.size() == 1 ? max_streams : (max_streams * region_num + total_local_region_num - 1) / total_local_region_num;
-
+        size_t current_max_streams = max_streams;
```

Comment:
Seems we don't need `current_max_streams` anymore, just use `max_streams` is enough.

Reply:
done